src/django_charting/templatetags/charts.py

```diff
@@ -0,0 +1,10 @@
+from django import template
+
+register = template.Library()
+
+
+@register.inclusion_tag("charts/chart.html")
+def render_chart(chart):
+    return {
+        "chart": chart,
+    }
```

Comment:
exact gelijke implementatie in `charting.py` en `charts.py`. Zou niet 1 van de twee een shim moeten zijn om de andere die een deprecation warning geeft?

Reply:
Ik stel voor de templatetags ook `django_charting` te noemen.

Reply:
prima!